src/Renderer/LayeredMaterial.js

```diff
@@ -57,6 +57,14 @@ var moveElementArray = function moveElementArray(array, oldIndex, newIndex) {
     array.splice(newIndex, 0, array.splice(oldIndex, 1)[0]);
 };
 
+const maxSamplersColorCount = 15;
+const samplersElevationCount = 1;
+
+export function getMaxColorSamplerUnitsCount() {
```

Comment:
Maybe it could move to `Capabilities` ?

Reply:
Looks ok: Already in cap for the test `Capabilities.getMaxTextureUnitsCount() `
` maxSamplersColorCount = 15` is specific to layeredMaterial